src/daffodilDebugger/utils.ts

```diff
@@ -33,32 +32,45 @@ export const daffodilArtifact = (version: string): Artifact => {
   return new Artifact('daffodil-debugger', version, 'daffodil-debugger')
 }
 
-export async function buildDebugger(baseFolder: string, filePath: string) {
-  if (!filePath.includes('.vscode/extension')) {
-    if (!fs.existsSync(filePath)) {
-      let command = osCheck(
-        'sbt universal:packageBin',
-        '/bin/bash --login -c "sbt universal:packageBin"'
-      ) // Needed --login so it could resolve sbt command
+export async function buildDebugger(
```

Comment:
Concern: this function seems misplaced (within the extension itself). Couldn't the `sbt` commands be added to the "extensionHost" launch config, so a local launch of the extension would (optionally) ensure the local debugger backend was updated?

Reply:
So we do have something like that already, however since we run `yarn watch` after the first the extension is ran the `sbt` command will get ran but after it won't unless you kill the `yarn watch` and relaunch the extension.

Reply:
Hmm/grrr. I will think more, but I still think this should be in build-space, not extension-space.

Reply:
@arosien @mbeckerle There is a node package called `nodemon` where I can add a custom watcher for scala,sbt related files and when they are changed run a `sbt universal:packageBin` and then could remove that from the extension. Would that be preferred in this PR?

Reply:
I am going to add it because `nodemon` will only be a dev dependency and makes debugging much easier. I was able to get to watch all the files related to the debugger files